### ISSUE TITLE
react-clipboard.js@2.0.4 issues

### ISSUE THREAD

**Issue by LennyLip** [nihey/react-clipboard.js#66]
First of all:
react-clipboard.js
and
react-clipboard.min.js
are minified both. 
3266 bytes.
And I get an error with minified version (using webpack)

```
ERROR in ./~/react-clipboard.js/dist/react-clipboard.js
  Module parse failed: /opt/python/bundle/153/app/node_modules/react-clipboard.js/dist/react-clipboard.js Unexpected token (1:2333)
```

**Comment by nihey** [comment]
@LennyLip Thanks for reporting, I'll investigate this immeadiately

**Comment by nihey** [comment]
@LennyLip I've just fixed it and released `2.0.5`, could you try out with this version? It should fix it.

As I've seen, this issue may be felt anyone who does not use Babel on development, with the latest change, I've made it so `react-clipboard.js` babel conversion supports more browsers.

The `dist/react-clipboard.js` file being minified issue is now fixed too.
